### PR TITLE
docs(SRV): rework transform images tutorial MTA-4240

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Comment by SamyOubouaziz** [comment]
Waiting for specs

File: tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx [hunk 1]
```diff
@@ -15,30 +15,18 @@ dates:
...
-## Introduction
-
-This tutorial is the first of a series of two. You will learn how to set up your environment, then [how to create and deploy functions](/tutorials/transform-bucket-images-triggers-functions-deploy/) to resize images stored in a bucket and push them to another bucket.
-
-<Lightbox src="scaleway-faas-image-transform.webp" alt="" />
-
 Serverless Functions are an asynchronous microservices architecture where event sources are decorrelated from event consumers.
 
-They work best when they are triggered by events stored in queues, which means that they can be edited without having to modify every microservice in the process.
+They work best when they are triggered by specific events, such as cron schedules, which means that they can be edited without having to modify every microservice in the process.
```
Comment (jcirinosclwy) [inline tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx:20]:
```suggestion
They work best when they are triggered by specific events, such as cron schedules, which means they can be edited without having to modify every microservice in the process.
```

File: tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx [hunk 2]
```diff
@@ -15,30 +15,18 @@ dates:
...
-In this tutorial, you will create the following resources to use your functions:
-
-- A project dedicated to this tutorial
-- Two buckets to store media
-- A queue to manage events
-- IAM credentials to secure your environment
-- Cockpit to monitor your functions
+In this tutorial, we will deploy a function that manipulates images stored in a source bucket and push them to another bucket using the [AWS SDK for Javascript](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/s3/).
 
 <Macro id="iam-requirements" />
```
Comment (jcirinosclwy) [inline tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx:24]:
```suggestion
<Macro id="requirements" />
```

File: tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx [hunk 3]
```diff
@@ -15,30 +15,18 @@ dates:
...
  
 <Message type="requirement">
```
Comment (jcirinosclwy) [inline tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx:26]:
```suggestion
```

File: tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx [hunk 4]
```diff
@@ -15,30 +15,18 @@ dates:
...
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
```
Comment (jcirinosclwy) [inline tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx:27]:
```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com)
```

File: tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx [hunk 5]
```diff
@@ -15,30 +15,18 @@ dates:
...
-  - You have around ten JPG or PNG images.
-  - You have a code editor installed.
+  - You have [NodeJS](https://nodejs.org/en/download) and [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) installed.
```
Comment (jcirinosclwy) [inline tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx:28]:
```suggestion
  - Installed [Node.js](https://nodejs.org/en/download) and [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
```

File: tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx [hunk 6]
```diff
@@ -15,30 +15,18 @@ dates:
...
+  - You have at least ten JPG or PNG images.
```
Comment (jcirinosclwy) [inline tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx:29]:
```suggestion
  - At least ten JPG or PNG images
```

File: tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx [hunk 7]
```diff
@@ -15,30 +15,18 @@ dates:
...
 </Message>
 
 ## Creating a New Project
```
Comment (jcirinosclwy) [inline tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx:32]:
```suggestion
## Creating a Project
```

File: tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx [hunk 8]
```diff
@@ -15,30 +15,18 @@ dates:
...
 
 <Macro id="iam-requirements" />
  
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
-  - You have around ten JPG or PNG images.
-  - You have a code editor installed.
+  - You have [NodeJS](https://nodejs.org/en/download) and [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) installed.
+  - You have at least ten JPG or PNG images.
 </Message>
```
Comment (jcirinosclwy) [inline tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx:30]:
```suggestion
```

**Review by jcirinosclwy — APPROVED** [review]

**Review by derkens-scw — APPROVED** [review]
RAS

**Review by thomas-tacquet — APPROVED** [review]

**Pull request merged**